src/main/java/com/cag/cagbackendapi/daos/impl/ProfileDao.kt

```diff
@@ -74,4 +92,25 @@ class ProfileDao : ProfileDaoI{
     private fun profileDtoToEntity(profileDto: ProfileDto): ProfileEntity {
         return objectMapper.convertValue(profileDto, ProfileEntity::class.java) //.map(profileDto, ProfileEntity::class.java)
     }
+
+    //user should not be able to create a new
+    private fun getUnionStatusEntity(demographicUnionStatus: String?): UnionStatusEntity {
+        return if (unionStatusRepository.getByName(demographicUnionStatus) != null ) {
+            unionStatusRepository.getByName(demographicUnionStatus)
+        } else {
+            var unionStatusEntity = UnionStatusEntity(null, demographicUnionStatus)
+            unionStatusRepository.save(unionStatusEntity)
```

Comment:
This is my fault for not detailing the stories. We do not want to allow users to add new union statuses. They cannot add new ones, they can only select from what is already there. I'm going to detail all the stories right now - my bad!!!